File: Lua/bee 1015.lua
local x1 = tonumber(io.read("*n"))
local y1 = tonumber(io.read("*n"))
local x2 = tonumber(io.read("*n"))
local y2 = tonumber(io.read("*n"))

print(string.format("%.4f", (math.sqrt((x2 - x1)^2 + (y2-y1)^2))))

File: Ruby/bee 1015.rb
x1, y1 = gets.split(" ").map{|x|  x.to_f}
x2, y2 = gets.split(" ").map {|x| x.to_f}

puts "%.4f" % Math.sqrt((x2 - x1) ** 2 + (y2 - y1) ** 2)
